clang/include/clang/Basic/TypeTraits.h

```diff
@@ -27,7 +27,8 @@ enum TypeTrait {
   ,
 #define TYPE_TRAIT_2(Spelling, Name, Key) BTT_##Name,
 #include "clang/Basic/TokenKinds.def"
-  BTT_Last = UTT_Last // BTT_Last == last BTT_XX in the enum.
+  BTT_Last = UTT_Last
```

Comment:
Why remove the comment?

Reply:
oops, this is not intended. This was left when I reverted some changes.

Reply:
Reverted the change.